api/envoy/extensions/filters/network/zookeeper_proxy/v3/zookeeper_proxy.proto

```diff
@@ -59,6 +59,9 @@ message ZooKeeperProxy {
   // threshold.
   // Specifying latency threshold overrides multiple times for one opcode is not allowed.
   repeated LatencyThresholdOverride latency_threshold_overrides = 6;
+
+  // Whether to emit per opcode request/response bytes metrics. If not set, defaults to false.
+  bool enable_per_opcode_request_response_bytes_metrics = 7;
```

Comment:
Do you think there will be a benefit to emit just the request or the response metrics?
If so, I suggest splitting it to 2 fields.